### ISSUE TITLE
View / edit address before opening letters & docs - Accessibility issues

### ISSUE THREAD

**Issue by esther-k** [department-of-veterans-affairs/va.gov-team#16491]
## What Happened?
**VoiceOver / TalkBack**

1. On Android and iOS, the a11yHint for the cancel and save buttons are either not being picked up by the screen reader or are picked up incorrectly
     - NOTE: Should be updated with [a11y update ticket](https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/17185)
2. Save button accessibility bug on iOS is covered in a [separate ticket ](https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/16395)
     - See # 3

## Specs:
- Device: Android - Google Pixel 3XL with TalkBack enabled
- Device: iOS - iPhone XR with VoiceOver enabled

## Steps to Reproduce
**VoiceOver / TalkBack**
For 1:
1. With Screen reader enabled on Android or iOS, navigate to the letter overview page 
2. Select the Mailing Address "button"
3. Select the cancel button and note that the a11yHint is not read 
4. Fill out the required fields 
5. Select the save button. Note that the a11yHint "saves changes" is read 

## Desired Behavior
**VoiceOver / TalkBack**
 
   - NOTE: Should be updated with [a11y update ticket](https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/17185) 🔽

For 1:
- The cancel button a11yHint should be picked up as “Cancels changes and navigates to the previous page”
- The save button a11yHint should be picked up as "Saves changes and navigates to the previous page”

## Acceptance Criteria
- See desired behavior

## Bug Severity
- 4 - Low

## Linked Story (s)
- Story #13940

## Screen Shot (s) and Additional Information

**Comment by digitalken** [comment]
PASS 2/17/2021 - Cancel and Back button hints are as expected.  Closing.
